src/EA.Weee.Web/Areas/Producer/Views/ProducerSubmission/AppropriateSignatory.cshtml

```diff
@@ -51,6 +50,11 @@
             @Html.Gds().TextBoxFor(m => m.Contact.Position, new { @maxlength = CommonMaxFieldLengths.Position })
         </div>
 
+        <h2 class="govuk-heading-s">Now send your application</h2>
+
+        <div class="govuk-body">By submitting this application you are confirming that, to the best of your knowledge, the details you are providing are correct.</div>
```

Comment:
@stevelee-civica  use P tags rather than divs in this view.

Reply:
updated